svgo.config.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-env node, es6 */
```

Comment:
Do we _need_ to put this file at the root of the repo?  The root dir is already quite cluttered and this filename does not start with a . so it is not hidden in directory listings.  Could this file live in conf / config or some other directory or _must_ it live at the root?

Reply:
Agreed on all points @cclauss - there is nothing preventing that script residing in /conf/config. I will update the PR. Thank you.

Reply:
/config is an alias that points to /conf so put the file in /conf and then look for it in either directory.